dockstore-common/src/main/java/io/dockstore/common/DescriptorLanguageSubclass.java

```diff
@@ -0,0 +1,55 @@
+/*
+ *
+ *    Copyright 2020 OICR
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ *
+ */
+
+package io.dockstore.common;
+
+import java.util.Arrays;
+import java.util.Optional;
+
+public enum DescriptorLanguageSubclass {
+    DOCKER_COMPOSE("docker-compose"),
+    HELM("helm"),
+    SWARM("swarm"),
+    KUBERNETES("kubernetes"),
+    NOT_APPLICABLE("n/a");
```

Comment:
I wonder if we shouldn't have OTHER as well. The service definition is pretty agnostic to the technology. But I'm not sure what anybody would do with OTHER, so I'm torn.

Reply:
I think for now we should limit service support to just ones we have blessed. Once it becomes more mature or we get requests we could investigate adding more.